### PR TITLE
Exchange Publication Plugin Integration

### PR DESCRIPTION
**Pull request by marianomerlo**

### CODE REVIEW

File: mule-artifact-it/mule-deployer-it/pom.xml
Comment (marianomerlo) [inline mule-artifact-it/mule-deployer-it/pom.xml:6]: Do not panic. We will rollback this before merging. We had to stop depending on SNAPSHOTS.

File: pom.xml
Comment (marianomerlo) [inline pom.xml:50]: Do not panic. We will rollback this before merging. We had to stop depending on SNAPSHOTS.

**Review by julianpascualMuleSoft — APPROVED** [review]

**Comment by julianpascualMuleSoft** [comment]
LGTM... Just please remember to roll back all the versions changes before merging it.

**Comment by marianomerlo** [comment]
@julianpascualMuleSoft You may want to give another review (of the last two commits), please 🙏 . 

Thank you in advance.

**Review by julianpascualMuleSoft — APPROVED** [review]

**Pull request merged**
